swagger.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Can we use the bearer for the authentication and authorization

Reply:
Also remember to persist the token

Reply:
Thanks for the feedback, it is done.